scss/inc/_base.scss

```diff
@@ -420,12 +497,24 @@ body {
     // added to make _u_ in ck QTI compatible
     .txt-underline {
         text-decoration: underline;
-        font-size : inherit;
+        font-size: inherit;
+    }
+    .writing-mode-vertical-rl .txt-underline {
+        text-underline-position: right;
+    }
+    .writing-mode-vertical-rl[data-useragent-browser='safari'] .txt-underline {
+        // for Safari < 18.2 which doesn't support "text-underline-position: right"
+        text-decoration: none;
+        border-right: 1px solid currentColor;
+    }
+
+    .txt-combine-upright-all {
```

Comment:
Is this class name ok with you?

Customers also want to use this in the authored text, so I'll suggest them to use this class. (or they can use item stylesheets; they agreed to edit html using CkEditor SourceDialog plugin)

Reply:
It's fine! (To be honest I had no idea what it meant, until I looked up [text-combine-upright](https://developer.mozilla.org/en-US/docs/Web/CSS/text-combine-upright).) I think no-one will ever use it but Japanese or Chinese authors, so a really specific name is fine.